src/modules/MainNav.pug

```diff
@@ -0,0 +1,10 @@
+doctype html
```

Comment:
This file should not contain a full structured document with html, head, body. Instead, here you should just have the portion of the MainNav. For the Setup itself, this is not a problem but it's better if you change it now so you can avoid get confused later by this.